lib/logstash/outputs/elasticsearch.rb

```diff
@@ -403,7 +403,7 @@ def register
       :upsert => @upsert,
       :doc_as_upsert => @doc_as_upsert
     }
-    common_options.merge! update_options if @action == 'update'
+    common_options.merge! update_options
```

Comment:
I'm curious as to the reason for this change. I would assume adding in these two settings might cause some issues no?